src/test/RUNTESTS

```diff
@@ -127,6 +127,7 @@ runtest() {
 				[ $retval = 124 -o $retval = 137 ] && errmsg='timed out'
 
 				[ $retval != 0 ] && {
+					[ -t 1 ] && errmsg="\033[31m$errmsg\033[m"
```

Comment:
2 reasons: laziness & reluctance to spawning two more processes for each PASS.

I think all modern terminals should accept ANSI escape codes and among nvml developers this number should be 100% ;)

But I can use tput if you'll insist.